app/controllers/iiif_controller.rb

```diff
@@ -71,7 +71,7 @@ def set_image_response_headers
   end
 
   def set_attachment_content_disposition_header
-    response.headers['Content-Disposition'] = "attachment;filename=#{params[:identifier]}.#{params[:format]}"
+    response.headers['Content-Disposition'] = "attachment;filename=#{identifier_params[:file_name]}.#{params[:format]}"
```

Comment:
My understanding was the identifier always included "druid/" in our case, so it **should** always work.  I wasn't able to identify a more direct way of fetching the filename from the params without going through the identifier_params method (which does rely on splitting on the encoded / character).  Can you see a way of avoiding this roundabout method of getting the filename?